src/rest-server/config/rest-server.yaml

```diff
@@ -32,3 +32,9 @@ github-path: marketplace
 debugging-reservation-seconds: 604800
 enable-priority-class: "true"
 sql-max-connection: 50
+# `hived-computing-device-envs` only works when hived scheduler is enabled.
+# It determines the environmental variables we inject into the container.
```

Comment:
environmental -> environment?
add a reference to https://github.com/microsoft/hivedscheduler/blob/master/doc/user-manual.md#scheduling-gpus

Reply:
done